package.json

```diff
@@ -4,28 +4,23 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "webpack-dev-server --open",
-    "build": "webpack"
+    "test": "jest"
+  },
```

Comment:
- [OPTIONAL] Add a `--coverage` flag to your test script in package.json. That way you will see the percentage coverage of your application code every time you run a test:
```
"scripts": {
        "test": "jest --coverage",
        ...
    }
```